workspaces/arborist/lib/arborist/reify.js

```diff
@@ -243,6 +243,7 @@ module.exports = cls => class Reifier extends cls {
       for (const actual of this.actualTree.inventory.values()) {
         if (actual.ideallyInert) {
           const ideal = this.idealTree.inventory.get(actual.name)
+          /* istanbul ignore else */
```

Comment:
once I made the change to `name` above, I wasn't able to make it _not_ work

Reply:
add a comment to the code itself as to why this is ignored please

Reply:
https://github.com/npm/cli/pull/8184/commits/b96a5cbc618206416dd683c6269a2b1f02af9065

Reply:
https://github.com/npm/cli/pull/8184/commits/f635c978a3045bcb35322717c656a02e2ceafeb4

even better, doing things the correct way means I could get rid of it and have coverage